frontend-react/src/pages/Upload.tsx

```diff
@@ -19,6 +21,57 @@ import {
 import { showError } from "../components/AlertNotifications";
 import Spinner from "../components/Spinner";
 
+const TransitionBanner = () => {
+    return (
+        <Alert
+            type="info"
+            heading="CVS uploader will be moving to SimpleReport"
+            headingLevel="h4"
+        >
+            <ul>
+                <li className="margin-y-2">
+                    Starting <b>July 18, 2022</b>, the CSV uploader feature will
+                    no longer be available from the ReportStream web portal and
+                    will be moving to our sister product, SimpleReport.
```

Comment:
Could we hyperlink SimpleReport with their website: https://www.simplereport.gov/